terraform/modules/daos_server/templates/pool_cont_create.tftpl

```diff
@@ -0,0 +1,58 @@
+%{ if length(pools) != 0 }
+set -x
+
+echo "BEGIN: DAOS pool and container creation"
+
+if [[ "$${HOSTNAME,,}" == "$${FIRST_DAOS_SERVER_HOSTNAME,,}" ]]; then
+  # Wait for servers to start
+  until dmg network scan | grep --fixed-strings "${servers}"
+  do
+    sleep 5
+  done
+  echo "All DAOS Servers started"
+
+  echo "Formatting storage on servers: ${servers}"
+  dmg storage format
+  dmg system query -v
+  echo "Done formating DAOS server"
+
```

Comment:
Only the below if for creating pools and containers. The above sections is for "formatting" the servers.   Likely the above should go back into the server startup section.

Reply:
I have a patch in progress to put this into a daos_server_format.

Reply:
Storage format is separated now.